src/App.js

```diff
@@ -0,0 +1,108 @@
+// import "./App.scss";
+import "./styles/main.scss";
+import Header from "./components/Header";
+import AddNewInput from "./components/AddNewInput";
+import Filter from "./components/FilterSortItems";
+import CheckListItem from "./components/CheckListItem";
+import React, { Component } from "react";
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newTodo: "",
+      createdDate: "",
+      items: [
+        {
+          task: "go to shopping",
+          createdDate: "25 May 2021",
+          status: "pending",
+        },
+      ],
+    };
+  }
+
+  handleOnChange = (event) => {
+    this.setState({ newTodo: event.target.value });
+  };
+  handleDate = (date) => {
+    console.log("date", date, new Date(date));
+    this.setState({ createdDate: date.toDateString() });
+  };
+
+  handleOnSubmit = (event) => {
+    event.preventDefault();
+    console.log("task", this.state.newTodo, this.state.createdDate);
```

Comment:
remove the unnecessary console.logs, there seems to be too many in your code. It might be confusing for you.